examples/event_handling/EventHandling.cpp

```diff
@@ -335,4 +383,5 @@ int main()
 {
     Application application;
     application.run();
+    return 0;
```

Comment:
C++ doesn't require a return statement in the main method. I wouldn't add it, or did you have a specific reason to do so?

Reply:
Because we hijack the main function, so the exception doesn't apply to our hijacked method and it requires a return value